number/src/serialize.rs

```diff
@@ -74,6 +74,10 @@ pub fn read_polys_csv_file<T: FieldElement>(file: impl Read) -> Vec<(String, Vec
     for result in reader.records() {
         let record = result.unwrap();
         for (idx, value) in record.iter().enumerate() {
+            // shorter polys/columns end in empty cells
```

Comment:
wdym shorter here? sounds like it's just empty in the end?

Reply:
with VADCOP, the cells for rows larger than a machine's degree are empty

Reply:
But then the entire col isnt written?